app/src/routes/(auth)/manager/structures/[structure_id]/+page.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="ts">
+	import Breadcrumbs from '$lib/ui/base/Breadcrumbs.svelte';
+	import { operationStore, query } from '@urql/svelte';
+	import { GetStructureByIdDocument, RoleEnum } from '$lib/graphql/_gen/typed-document-nodes';
+	import StructureEditLayer from '$lib/ui/StructureEdit/StructureEditLayer.svelte';
+	import LoaderIndicator from '$lib/ui/utils/LoaderIndicator.svelte';
+	import type { PageData } from './$types';
+	import { homeForRole } from '$lib/routes';
+
+	export let data: PageData;
+
+	const getStructure = operationStore(GetStructureByIdDocument, { structureId: data.structureId });
+	query(getStructure);
+
+	$: structure = $getStructure.data?.structure_by_pk;
+
+	$: breadcrumbs = [
+		{
+			name: 'Liste des structures',
+			path: `${homeForRole(RoleEnum.Manager)}/structures`,
+			label: 'Liste des Structures',
+		},
+		{
+			name: 'Mettre à jour les informations de structure',
+			path: `${homeForRole(RoleEnum.Manager)}/${data.structureId}`,
```

Comment:
J'ai l'impression que ça n'a aucun impact sur le rendu… mais ce `path` me paraît incorrect.

```suggestion
			path: `${homeForRole(RoleEnum.Manager)}/structures/${data.structureId}`,
```

Peut-être qu'il vaut mieux ne pas le mettre ?

```suggestion
```

D'ailleurs je ne vois pas à quoi sert la propriété `name` dans ce cas non plus…

Reply:
ah oui bien vu, pour le dernier élément des _miettes de pain_ ca ne sert à rien